.github/workflows/build.yml

```diff
@@ -74,6 +74,6 @@ jobs:
             - name: Set output variables
               id: bake_metadata
               run: |
-                  .github/workflows/extract-image-names.sh | tee -a "${GITHUB_OUTPUT}"
+                  .github/workflows/extract-image-names.sh | tee -a "${GITHUB_OUTPUT}" | awk -F'=' '{print $2}' | jq
```

Comment:
This I don't understand, what different it will make?

Reply:
Oh, this just pretty prints the json output so that it is easier to read in the Github Actions logs. Previously it was on a single line, and it was a bit annoying when I needed to copy the image name.

Reply:
This is how it looks now in the amd64 image build

![image](https://github.com/aiidalab/aiidalab-docker-stack/assets/9539441/c9a2609c-d537-4dae-ad0d-773fced381bc)